7_react/react1/week1/react1/src/App.js

```diff
@@ -0,0 +1,38 @@
+import './App.css';
+
+function App() {
+  return (
+    <div className="React App">
+      <ToDoList />
+    </div>
+  );
+}
+
+export default App;
+
+const todo = [
+  { activity: 'Get out of bed', date: 'Wed Sep 13 2017' },
+  { activity: 'Brush teeth', date: 'Thu Sep 14 2017' },
+  { activity: 'Eat breakfast', date: 'Fri Sep 15 2017' },
+];
+
+
+function ToDoList() {
+  return (
+    <div>
+      {todo.map((todo) => {
+        const date = todo.date;
+        const activity = todo.activity;
+        return <ToDoItem date={date} activity={activity} />;
+
+      })}
+    </div>
```

Comment:
Here you have two variables with the same name:
* `todo` the array of data
* `todo` the item being iterated in the `map` function

It can easily become confusing for a developer to read such code. Consider to give unique names to all variables to make it easier to distinguish them from each other. A common practice is to give array type variables a plural name like `todos`. This makes it clear that it is an array that can contain multiple items. Each item you can then refer to as `todo`